diff --git a/lib/vendor/symfony/lib/helper/PartialHelper.php b/lib/vendor/symfony/lib/helper/PartialHelper.php
index e20d0b9..ec1a846 100644
--- a/lib/vendor/symfony/lib/helper/PartialHelper.php
+++ b/lib/vendor/symfony/lib/helper/PartialHelper.php
@@ -215,7 +215,7 @@ function get_partial($templateName, $vars = array())
 
   $class = sfConfig::get('mod_'.strtolower($moduleName).'_partial_view_class', 'sf').'PartialView';
   $view = new $class($context, $moduleName, $actionName, '');
-  $view->setPartialVars(true === sfConfig::get('sf_escaping_strategy') ? sfOutputEscaper::unescape($vars) : $vars);
+  $view->setPartialVars($vars);
 
   return $view->render();
 }
